test/resources/Subscriptions.spec.js

```diff
@@ -126,7 +126,7 @@ describe('subscriptions Resource', () => {
     it('Sends the correct request', () => {
       stripe.subscriptions.update('test_sub', {
         items: {
-          '0': {
+          0: {
```

Comment:
This seems like an actual functionality change - `'0'` isn't the same as `0`. Let's double check what the API returns here and fix this to match that. If it's actual number indices, should this be an array?

Reply:
Reverted changes to this file.